content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -501,6 +501,36 @@ As this brings Bevy closer to full support of Android, there isn't a need anymor
 
 ![iOS emulator running Bevy](ios%20emulator.png)
 
+## Enable Parallel Pipelined Rendering
+
+<div class="release-feature-authors">authors: @hymm, @james7132</div>
+
+![Trace with Pipelined Rendering](pipelined-rendering-trace.png)
+
+On multithreaded platforms bevy will now run significantly faster by running simulation and
+rendering in parallel. The renderer was rearchiteched in [bevy 0.6](https://bevyengine.org/news/bevy-0-6/#pipelined-rendering-extract-prepare-queue-render)
+to enable this, but the final step of actually running them in parallel was not done until
+now. There was a bit of tricky work to figure out around enabling running specific rendering
+systems on the main app thread instead of the rendering thread and making sure !Send
+resources aren't accessed from a different thread than they are initialized on.
```

Comment:
I found this sentence kinda hard to parse. Maybe something like this?

```suggestion
now. There were some complications when moving specific rendering
systems from the rendering thread to the main app thread, and making sure !Send
resources aren't accessed from a different thread than they are initialized on.
```

Reply:
tried to add more sentences instead. Does it read better now?

Reply:
Yes it does, though I wonder if it's going into too much detail about the implementation now. Not 100% sure what level of detail is intended for this post